extension/package.json

```diff
@@ -5,14 +5,13 @@
   "license": "MIT",
   "type": "module",
   "scripts": {
-    "check-coverage": "nyc check-coverage --lines 89",
-    "test": "npm run check && npm run cover:unit && npm run cover:integration && npm run cover:e2e && npm run check-coverage",
-    "cover:unit": "nyc --silent npm run unit",
-    "cover:integration": "nyc --silent --no-clean npm run integration",
-    "cover:e2e": "nyc --silent --no-clean npm run e2e",
-    "unit": "nyc mocha --file ./test/unit/mock-default-confg.js \"test/unit/**/*.spec.js\"",
-    "integration": "nyc mocha --file ./test/initial-integration-test-set-up.js \"test/integration/**/*.spec.js\"",
-    "e2e": "nyc mocha --file ./test/initial-integration-test-set-up.js \"test/e2e/**/*.spec.js\"",
+    "test-coverage-run": "c8 npm run test-run && npm run check-coverage",
+    "test-run": "npm run unit && npm run integration && npm run e2e",
+    "check-coverage": "c8 check-coverage --lines 89",
```

Comment:
This command alone doesn't make sense. Please try to remove it and update `test-coverage-run`.

Reply:
I found it more reasonable to have `cover:unit` etc. for such case than to have a command that cannot be run by itself.

Reply:
cover:unit is a different thing, but keeping this command `check-coverage` is also for refactoring to make it more clear for reading. 
So I would keep it this way.